cpp/tests/constraints.cpp

```diff
@@ -30,8 +58,38 @@ TEST(Constraints, LinearJointVelocity) {
     EXPECT_EQ(F .size(), 0);
     EXPECT_EQ(g .size(), 0);
     EXPECT_EQ(ub.size(), 0);
-    EXPECT_EQ(xb.size(), N+1);
+    EXPECT_EQ(xb.size(), N);
     EXPECT_NO_THROW(ljv.computeParams(path, gridpoints, a, b, c, F, g, ub, xb));
   }
 }
 
+#ifdef BUILD_WITH_PINOCCHIO
+TEST_F(Constraint, jointTorquePinocchio) {
+  using namespace toppra;
+  typedef constraint::jointTorque::Pinocchio<> JointTorque;
+
+  JointTorque::Model model;
```

Comment:
Is it a default empty model? It would be nice to have a test that demonstrates how to use the class on a real robot.

Reply:
This creates an empty model. It is then filled with an example model at line 72. There are some examples on how to load a real robot model with Pinocchio. E.g. see https://github.com/stack-of-tasks/pinocchio/blob/master/examples/overview-urdf.cpp